src/main/java/reactor/netty/http/client/HttpClientConnect.java

```diff
@@ -512,6 +468,7 @@ public void onStateChange(Connection connection, State newState) {
 			this.websocketProtocols = configuration.websocketSubprotocols;
 			this.maxFramePayloadLength = configuration.websocketMaxFramePayloadLength;
 			this.websocketProxyPing = configuration.websocketProxyPing;
+			this.retried = !configuration.retryEnabled;
```

Comment:
I think this would be easier to read as `shouldRetry`.

Reply:
done